docs/language-server.markdown

```diff
@@ -207,6 +210,9 @@ Supported settings and their defaults. See information for your language server
 
 ```json
 {
+  // A suggestion for the formatter about how wide (in columns) to print definitions.
+  "formattingWidth": 80,
+
   // The number of completions the server should collect and send based on a single query.
   // Increasing this limit will provide more completion results, but at the cost of being slower to respond.
   // If explicitly set to `null` the server will return ALL completions available.
```

Comment:
I guess comments aren't legal in json? I know you didn't introduce them, and it should technically go into a separate PR, but since our eyes are here...
```suggestion
* `formattingWidth`: A suggestion for the formatter about how wide (in columns) to print definitions.
* `maxCompletions`: The number of completions the server should collect and send based on a single query.  Increasing this limit will provide more completion results, but at the cost of being slower to respond.
If explicitly set to `null` the server will return ALL completions available.

```json
{
  "formattingWidth": 80,
```

Reply:
It looks Unison uses a non-squashing workflow, so I don’t have a problem with this being in the same PR (so long as the commits stay separate).

Reply:
Ok, well, after applying your suggestion, I amended the commit to make the second paragraph of `maxCompletions` doc part of the list item. I _thought_ it would keep you as the author, but it didn’t (although either git or Magit at least added a `Co-authored-by` field).

I also tried to find a way to edit a suggestion before committing, but doesn’t seem like there is one.

Anyway, just some Git(Hub) bikeshedding. Hopefully not a big deal.